tutorials/physics/interpolation/advanced_physics_interpolation.rst

```diff
@@ -0,0 +1,173 @@
+.. _doc_advanced_physics_interpolation:
+
+Advanced physics interpolation
+==============================
+
+Although the previous instructions will give satisfactory results in a lot of games,
+in some cases you will want to go a stage further to get the best possible results
+and the smoothest possible experience.
+
+Exceptions to automatic physics interpolation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Even with physics interpolation active, there may be some local situations where
+you would benefit from disabling automatic interpolation for a
+:ref:`Node<class_Node>` (or branch of the :ref:`SceneTree<class_SceneTree>`), and
+have the finer control of performing interpolation manually.
+
+This is possible using the :ref:`Node.physics_interpolation_mode<class_Node_property_physics_interpolation_mode>`
+property which is present in all Nodes. If you for example, turn off interpolation
+for a Node, the children will recursively also be affected (as they default to
+inheriting the parent setting). This means you can easily disable interpolation for
+an entire subscene.
+
+The most common situation where you may want to perform your own interpolation is
+Cameras.
+
+Cameras
+^^^^^^^
+
+In many cases, a :ref:`Camera3D<class_Camera3D>` can use automatic interpolation
+just like any other node. However, for best results, especially at low physics tick
+rates, it is recommended that you take a manual approach to Camera interpolation.
+
+This is because viewers are very sensitive to Camera movement. For instance, a
+Camera that realigns slightly every 1/10th of a second (at 10tps tick rate) will
+often be noticeable. You can get a much smoother result by moving the Camera each
+frame in ``_process``, and following an interpolated target manually.
+
+Manual Camera interpolation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Ensure the Camera is using global coordinate space**
```

Comment:
Is this supposed to be a header? If so you probably want to change this page to use a nested header structure rather than a flat one, see previous comment
![msedge_6J1JQdLVpo](https://github.com/user-attachments/assets/c1ea1d7e-fc19-47fb-b94a-487ce8b3f37d)

Reply:
I've changed it to a sub-sub header, and I've done the same for the headers below it related to cameras.